### ISSUE TITLE
SODLifeAndLiving

### ISSUE THREAD

**Issue by Wjoseph1** [Ven0maus/SOD.Common#31]
When loading into a save it freezes and does not load the save

it only happens when i have SODLifeAndLiving installed

this is my first time reporting a bug so i might be a little slow with everything.

I couldn't copy and paste it so i took a screenshot instead, sorry again i am not sure how to copy and paste from the little console window.

![Screenshot 2024-04-27 235132](https://github.com/Ven0maus/SOD.Common/assets/128953239/7fe0b7d4-8375-46fe-8bf9-d097d33511b0)

**Comment by b1u0312** [comment]
**Rolling back to version 1.1.8 of SODCommon resolved the issue for me.** Funnily enough this is also my first contribution to git, I'm not very technically literate but I hope this helps. Here it is in plaintext:
```
Exception: System.MissingMethodException: Method not found: 'System.String SOD.Common.Helpers.SaveGame.GetUniqueString(System.String)'.
   at SOD.LifeAndLiving.Plugin.SaveGame_OnBeforeSave(Object sender, SaveGameArgs e)
   at SOD.Common.Helpers.SaveGame.OnSave(String path, Boolean after)
   at SOD.Common.Patches.SaveStateControllerPatches.SaveStateController_CaptureSaveStateAsync.Prefix(String path)
   at DMD<SaveStateController::CaptureSaveStateAsync>(SaveStateController this, String path)
   at (il2cpp -> managed) CaptureSaveStateAsync(IntPtr , IntPtr , Il2CppMethodInfo* )
```

The same exception is also thrown when attempting to load or delete a save.
```
Exception: System.MissingMethodException: Method not found: 'System.String SOD.Common.Helpers.SaveGame.GetUniqueString(System.String)'.
   at SOD.LifeAndLiving.Plugin.SaveGame_OnBeforeLoad(Object sender, SaveGameArgs e)
   at SOD.Common.Helpers.SaveGame.OnLoad(String path, Boolean after)
   at SOD.Common.Patches.SaveStateControllerPatches.CityConstructor_StartLoading.Prefix(CityConstructor __instance)
   at DMD<CityConstructor::StartLoading>(CityConstructor this)
   at (il2cpp -> managed) StartLoading(IntPtr , Il2CppMethodInfo* )
```
```
Exception: System.MissingMethodException: Method not found: 'System.String SOD.Common.Helpers.SaveGame.GetUniqueString(System.String)'.
   at SOD.LifeAndLiving.Plugin.SaveGame_OnBeforeDelete(Object sender, SaveGameArgs e)
   at SOD.Common.Helpers.SaveGame.OnDelete(String path, Boolean after)
   at SOD.Common.Patches.SaveStateControllerPatches.MainMenuController_DeleteSave.Prefix(MainMenuController __instance)
   at DMD<MainMenuController::DeleteSave>(MainMenuController this)
   at (il2cpp -> managed) DeleteSave(IntPtr , Il2CppMethodInfo* )
```

**Comment by Wjoseph1** [comment]
Rolling back seemed to not help saldly ): thank you for the recommendation though

**Comment by Ven0maus** [comment]
Fixed in v2.0.0 of SOD.Common, sorry for the troubles :)

**Comment by Wjoseph1** [comment]
Thank you for your work! (:
